proto/babylonlabs/babylon/v1beta1/babylon.proto

```diff
@@ -12,16 +12,22 @@ option (gogoproto.equal_all) = false;
 message Params {
   option (gogoproto.equal) = true;
 
+  // babylon_contract_code_id is the code ID of the Babylon contract
+  uint64 babylon_contract_code_id = 1;
+  // btc_staking_contract_code_id is the code ID of the BTC staking contract
+  uint64 btc_staking_contract_code_id = 2;
+  // btc_finality_contract_code_id is the code ID of the BTC finality contract
+  uint64 btc_finality_contract_code_id = 3;
```

Comment:
Nit: I would just handle these internally / remove them from the `Params` query.